Vic2ToHoI4/Source/HOI4World/States/HoI4States.cpp

```diff
@@ -713,16 +713,28 @@ void HoI4::States::putIndustryInStates(const std::map<std::string, double>& fact
 	 const Configuration& theConfiguration)
 {
 	const StateCategories theStateCategories(theConfiguration);
+	std::vector<HoI4::State> sortedStates;
```

Comment:
Because this won't outlive `states`, you can instead make it `std::vector<HoI4::State&>`. That'll save the lookup of the original state later.